relay-sampling/src/lib.rs

```diff
@@ -441,9 +441,19 @@ impl FieldValueProvider for Event {
                 }
                 _ => Value::Null,
             },
+            "transaction.op" => match (self.ty.value(), store::get_transaction_op(self)) {
+                (Some(&EventType::Transaction), Some(op_name)) => Value::String(op_name.to_owned()),
+                _ => Value::Null,
+            },
+            "event.platform" => match self.platform.value() {
+                Some(platform) if store::is_valid_platform(platform) => {
+                    Value::String(platform.to_owned())
+                }
+                _ => Value::Null,
```

Comment:
Should this be `"other"`? Looking at https://github.com/getsentry/relay/blob/4298977c9e0231cf9f206c31b112ea234c28d4da/relay-general/src/store/normalize.rs#L492 as a source of truth.

Reply:
yup thanks